plugin/dca/dca.go

```diff
@@ -267,10 +269,178 @@ func (p *DCAPlugin) ProposeTransactions(policy types.PluginPolicy) ([]types.Plug
 }
 
 func (p *DCAPlugin) ValidateTransactionProposal(policy types.PluginPolicy, txs []types.PluginKeysignRequest) error {
-	p.logger.Debug("DCA: VALIDATE TRANSACTION PROPOSAL")
+	p.logger.Info("DCA: VALIDATE TRANSACTION PROPOSAL")
+
+	if len(txs) == 0 {
+		return fmt.Errorf("no transactions provided for validation")
+	}
+
+	hexChainCode := "1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef" // vault's chain code
```

Comment:
add a TODO, to not forget to handle this too as part of https://github.com/LimeChain/vultisig-plugins-marketplace/issues/82